dynamodb/src/main/scala/awscala/dynamodbv2/Table.scala

```diff
@@ -51,6 +55,51 @@ case class Table(
   def put(hashPK: Any, attributes: (String, Any)*)(implicit dynamoDB: DynamoDB): Unit = putItem(hashPK, attributes: _*)
   def put(hashPK: Any, rangePK: Any, attributes: (String, Any)*)(implicit dynamoDB: DynamoDB): Unit = putItem(hashPK, rangePK, attributes: _*)
 
+  def put[E <: AnyRef](entity: E)(implicit dynamoDB: DynamoDB):Unit = {
```

Comment:
Sorry, I should have mentioned this earlier. What do you think about having hashPK/rangePK as arguments rather than having the naming convention in the entity class? If I were a user of this method, I may want to use different names for the PK values in the class. So, just having them as args would be more flexible. Thoughts?

```suggestion
  def putEntity[E <: AnyRef](hashPK: Any, rangePK: Any, entity: E)(implicit dynamoDB: DynamoDB):Unit = {
```

Reply:
Actually, as a user, I was looking for a possibility to push the whole CC as a single argument :) 
To make you happy I have overloaded putItem() with hashPK and rangePK. Looking forward to your feedback!

Reply:
answering 2nd question.. we can potentially use annotations to give users freedom in naming PK in case class definition.

Reply:
Thanks for adding another method for it. It looks good to me.